srcs/utils/error_exit.c

```diff
@@ -15,7 +15,7 @@ t_error	error_and_exit(t_error code)
 	exit(EXIT_FAILURE);
 }
 
-t_error	error_and_return(t_error code, int exit_code)
+int	error_and_return(t_error code, int exit_code)

```

Comment:
oh ok so you did this on purpose. Ok !

Reply:
Sry 😅 I should have discussed this in your PR instead of just changing it, but I noticed it only after I merged

Reply:
haha no problem, I figured I just wasn't sure =)